src/components/ButtonGroup.tsx

```diff
@@ -17,9 +18,11 @@ export type ButtonGroupButton = {
   text?: string;
   onClick?: VoidFunction;
   /** Disables the button. Note we don't support the `disabled: ReactNode`/tooltip for now. */
```

Comment:
Thought: Wonder if this comment needs updated?

If I'm following, if you set _only_ disabled, it functions as a tooltip, but if you set both disabled and tooltip, then tooltip wins (and the button is still disabled)?

Reply:
I didn't even notice this comment. I will update it.
If you set `disabled={true} tooltip="Tooltip"`, then `tooltip` wins.
If you set `disabled="Disabled reason" tooltip="Tooltip"`, then `disabled` wins.

Reply:
Ah gotcha, sounds good!